game-app/game-core/src/main/java/games/strategy/triplea/odds/calculator/PlayerUnitsPanel.java

```diff
@@ -122,7 +124,8 @@ public void init(final GamePlayer gamePlayer, final List<Unit> units, final bool
 
     GamePlayer previousPlayer = null;
     JPanel panel = null;
-    for (final UnitCategory category : categories) {
+    unitCategories.sort(getComparatorUnitCategories(data));
```

Comment:
There is a sort of `unitCategories` higher up on line 69. Does this new sort call here, on line 127, overwrite the results of that previous sort?

Reply:
Good point. Another place where we can remove a "special" sorting with our generic one. Fix will be provided in commit #6.